src/test/java/io/stargate/sgv2/jsonapi/api/v1/RerankFeatureDisabledIntegrationTest.java

```diff
@@ -30,9 +30,10 @@ public TestResource() {}
 
     @Override
     public String getFeatureFlagReranking() {
-      // return empty to leave feature "undefined" (disabled unless per-request header override)
+      // return BLANK String to leave feature "undefined" (disabled unless per-request header
+      // override)
       // ("false" would be "disabled" for all tests, regardless of headers)
-      return "";
+      return " ";
```

Comment:
Using non-empty because apparently SmallRye fails on Empty String directly. What a piece-of-work library :-(